website/www/site/content/en/documentation/dsls/sql/extensions/create-external-table.md

```diff
@@ -473,18 +542,8 @@ Write Mode supports writing to a topic.
 *   CSV (default)
     *   Beam parses the messages, attempting to parse fields according to the
         types specified in the schema.
```

Comment:
Is there a jira for making CSV a PayloadSerializer?

Reply:
Not currently. CSV is weird in that one record transforms into multiple rows, which works weird with metadata (it would need to be duplicated, which may or may not make sense for headers if one of them, for example, is used for deduplication further down the line). I also have never heard of users shipping data on Pub/Sub-like systems (I count things like Kafka and Kinesis in this) in CSV format, it tends to be more used with file-based storage.

TLDR: we could, but the semantics would be weird and its probably of minimal value